CHANGELOG.rst

```diff
@@ -12,6 +12,9 @@ This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
 Added
 -----
+- Added the logging mechanism, to give a warning if the NLU picks up intent or entities
+  that are not exist in the domain. The logic of this warning is implemented in a function called
+  `_log_unseen_enitites` in rasa/core/processor.py.
```

Comment:
Would rephrase it a bit
```
- log a warning if the ``Interpreter`` picks up an intent or an entity that does not exist in the domain file.
```
No need to point to the function you added. Also no need for the empty line after the entry.